profile/src/main/java/rdx/works/profile/cloudbackup/data/DriveClient.kt

```diff
@@ -175,14 +174,15 @@ class DriveClientImpl @Inject constructor(
 
     override suspend fun claimCloudBackup(
         file: CloudBackupFileEntity,
-        updatedHeader: Header
+        claimingProfile: Profile
     ): Result<CloudBackupFileEntity> = withContext(ioDispatcher) {
         runCatching {
             Timber.tag("CloudBackup").d("Start claiming process with file id: ${file.id}")
             googleSignInManager.getDrive().files()
                 .update(
                     file.id.id,
-                    file.claim(header = updatedHeader) // Updates the lastUsedOnDevice.id
+                    file.claim(header = claimingProfile.header), // Updates the lastUsedOnDevice.id
+                    ByteArrayContent("application/json", claimingProfile.toJson().toByteArray())
```

Comment:
This is what we didn't used to do. Now update the metadata **AND** the content in the same call.